include/sm64.h

```diff
@@ -70,12 +70,48 @@ enum RenderLayers {
     LAYER_OCCLUDE_SILHOUETTE_OPAQUE,
     LAYER_OCCLUDE_SILHOUETTE_ALPHA,
 #endif
+    LAYER_CLD,
     LAYER_TRANSPARENT_DECAL,
     LAYER_TRANSPARENT,
     LAYER_TRANSPARENT_INTER,
     LAYER_COUNT
 };
 
+enum BatchOpaque {
+    BATCH_OPAQUE_CORKBOX,
+    BATCH_OPAQUE_COUNT,
+};
+
+enum BatchAlpha {
+    // coin frames batches, it goes 0, 1, 2, 3, 4, 3, 2, 1
+    BATCH_ALPHA_COINS_FIRST,
+    BATCH_ALPHA_COINS_LAST = BATCH_ALPHA_COINS_FIRST + 4,
```

Comment:
Idk how I feel about this, what does the 4 actually represent here, number of coin textures? If so, that should be defined somewhere, ideally automatically computed or enforced with a static assert if possible.

Ditto for the flames.

Reply:
I think the easiest solution is to add a static assert. `sm64.h` is a glue between subsystems geo->layer->batches->dls. It would be hard to deduce amount of coin textures without including actors in sm64.h which is not something I want to do.

For now I will remove those enums because they are unused anyways and reland them later with batching for specific objects.

Reply:
I have provided the static asserts for constant batches, it actually appeared to be pretty easy to implement.

Reply:
Could these be moved to `batch_list.h`? Idk if they make the most sense in `sm64.h` anyway, though to be fair the layer definitions are also here...